man/ostree.repo-config.xml

```diff
@@ -333,6 +333,41 @@ Boston, MA 02111-1307, USA.
 
   </refsect1>
 
+  <refsect1>
+    <title>[sysrooot] Section Options</title>
+
+    <para>
+      Options for the sysroot, which OSTree deploys the OSTree repository to.  The following entries are defined:
```

Comment:
How about "which contains the OSTree repository, deployments, and stateroots".

Reply:
:+1: SGTM, thanks